src/app/shared/services/claims-facade/claims-facade.service.ts

```diff
@@ -224,7 +224,7 @@ export class ClaimsFacadeService {
   }
 
   private async setIsRevokedOffChainStatus(enrolment: EnrolmentClaim) {
-    if (enrolment.isRevocableOffChain) {
```

Comment:
claims that were revoked off chain already were never getting into this logic to check their revocation status (and therefore to set isRevokedOffChain), so it was always coming back as true and therefore the button was always 'active'. In this way, we are just checking each credential with a statusListEntry.